src/lib/Libcmds/pbs_json.c

```diff
@@ -59,6 +59,75 @@ link_node(JsonNode *node) {
 
 /**
  * @brief
+ * Duplicates a string where the new string conforms to JSON
+ *
+ * @param[in] str - string to be duplicated
+ *
+ * @return string
+ * @retval JSON conforming string	success
+ * @retval NULL						error
+ *
+ */
+char*
+str_dup_json(const char *str)
```

Comment:
The C function name is strdup(), I'd drop the '_' between str and dup.  This function really doesn't have anything to do with json either.  Maybe call it strdup_escape().

Reply:
I used json because the original code only appeared in pbs_json.c. I'll change it